Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor

```diff
@@ -1,6 +1,416 @@
+            <li>position-bottom</li>
+        </ul>
+        <p>
+            Also be sure the off-canvas panel has a unique ID so it can be targeted by the click trigger.
+        </p>
+        <p>
+            Example:
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PE9mZkNhbnZhcyBQb3NpdGlvbj0icG9zaXRpdmUtbGVmdCIgSWQ9Im9mZkNhbnZhcyI+CjwvT2ZmQ2FudmFzPg==" />
+
+        <p>
+            Along with the Off-canvas component, the main content of your page will be housed in its own component with the component <Code Content="PE9mZkNhbnZhc0NvbnRlbnQ+"></Code>. This is where your page content lives.
+        </p>
+
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PE9mZkNhbnZhc0NvbnRlbnQ+CiAgPCEtLSBZb3VyIHBhZ2UgY29udGVudCBsaXZlcyBoZXJlIC0tPgo8L09mZkNhbnZhc0NvbnRlbnQ+" />
+
+        <p>
+            So putting it all together:
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PE9mZkNhbnZhcyBQb3NpdGlvbj0icG9zaXRpb24tbGVmdCIgSWQ9Im9mZkNhbnZhcyI+CiAgPCEtLSBZb3VyIG1lbnUgb3IgT2ZmLWNhbnZhcyBjb250ZW50IGdvZXMgaGVyZSAtLT4KPC9PZmZDYW52YXM+CjxPZmZDYW52YXNDb250ZW50PgogIDwhLS0gWW91ciBwYWdlIGNvbnRlbnQgbGl2ZXMgaGVyZSAtLT4KPC9PZmZDYW52YXNDb250ZW50Pg==" />
+
+        <h2>Wrapper</h2>
+        <p>
+            You can add an optional wrapper around off-canvas and the content. This hides the vertical (on top/bottom off-canvas) or horizontal (on left/right off-canvas) scrollbars the body will have. Simply use a component <Code Content="PE9mZkNhbnZhc1dyYXBwZXI+"></Code>
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PE9mZkNhbnZhc1dyYXBwZXI+CiAgPE9mZkNhbnZhcyBQb3NpdGlvbj0icG9zaXRpb24tbGVmdCIgSWQ9Im9mZkNhbnZhcyI+CiAgICA8IS0tIFlvdXIgbWVudSBvciBPZmYtY2FudmFzIGNvbnRlbnQgZ29lcyBoZXJlIC0tPgogIDwvT2ZmQ2FudmFzPgogIDxPZmZDYW52YXNDb250ZW50PgogICAgPCEtLSBZb3VyIHBhZ2UgY29udGVudCBsaXZlcyBoZXJlIC0tPgogIDwvT2ZmQ2FudmFzQ29udGVudD4KPC9PZmZDYW52YXNXcmFwcGVyPg==" />
+
+        <h2>Click Triggers</h2>
+        <p>
+            To create a click trigger that opens the panel, add the attribute <Code>DataToggle</Code> to component <Code>Button</Code> or <Code>data-open</Code> or  to any element. That element will then open or toggle the panel when clicked on. The value of the data attribute should be the ID of the off-canvas.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PEJ1dHRvbiBEYXRhVG9nZ2xlPSJvZmZDYW52YXMiPk9wZW4gTWVudTwvQnV0dG9uPg==" />
+        <Button DataToggle="offCanvas1">Open Menu</Button>
```

Comment:
Remove that button we dont need to see a button if doesn't do anything in that context.

Reply:
```suggestion
```